spec/features/mediation_table_spec.rb

```diff
@@ -197,6 +200,8 @@
         within('tbody td table tbody') do
           within(all('tr').last) do
             click_button('Approve')
+            expect(page).to have_button('Approve')
+            expect(page).to_not have_button('Approve', disabled: true)
```

Comment:
since the `disabled` attribute is removed to re-enable the button, i couldn't check for `disabled: false`, and instead checked for the presence of the button and the absence of the attribute.  originally, i did try checking `disabled: false`, since the attribute value in the HTML is actually set to `disabled`, and not a boolean value, but that caused a test failure, whereas this approach didn't, and manual testing indicated that re-enabling worked fine.